core/edit/edit.talon

```diff
@@ -161,6 +165,7 @@ paste match: edit.paste_match_style()
 (pace | paste) line: user.paste_line()
```

Comment:
Can we add an optional "to" to make the use of these clearer?

Reply:
done